scripts/run-android.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -m # needed to access jobs
```

Comment:
Would be nice to also use `set -euo pipefail`.

Reply:
addressed with https://github.com/status-im/status-mobile/pull/18784/commits/8a330cc8708d68b58e1ab943eb1f06dfe0c4e46d